Code/Tools/SceneAPI/SceneCore/Containers/SceneGraph.cpp

```diff
@@ -294,7 +294,7 @@ namespace AZ
                 node.m_parentIndex = parentIndex;
                 m_hierarchy.push_back(node);
 
-                AZ_Assert(IsValidName(name), "Name '%s' for SceneGraph sibling contains invalid characters", name);
+                //AZ_Assert(IsValidName(name), "Name '%s' for SceneGraph sibling contains invalid characters", name);
```

Comment:
I think this assert is valid. We'd need to have some kind of error correction for an invalid name if we remove the assert. Also, if we do remove the assert, we would remove the line not comment it out.

Reply:
`IsValidName` can return false if the char* is null, so even formatting the name here in the assert message is potentially UB.

I would think the `IsValidName` should become an if-check and add proper error handling.

Reply:
IMO in any case I don't think we should be commiting commented code.
Either keep it or remove it, but not comment it.